templates/tfengine/recipes/project.hcl

```diff
@@ -41,6 +41,14 @@ schema = {
         Can be one of the organization ID or folder ID according to parent_type.
       EOF
       type    = "string"
+      # TODO(ernestognw): Find an alternative to keep this in multi_envs
```

Comment:
It would be better if you create an Issue and link that in the TODO instead of your username.

Reply:
Done, defined at #987